mkdocs/docs/experiments/categories/pods/pod-network-corruption.md

```diff
@@ -170,7 +170,17 @@
         <td> DESTINATION_HOSTS </td>
         <td> DNS Names/FQDN names of the services, the accessibility to which, is impacted </td>
         <td> if not provided, it will induce network chaos for all ips/destinations or DESTINATION_IPS if already defined</td>
-      </tr>      
+      </tr>
+      <tr>
+        <td> SOURCE_PORTS </td>
+        <td> ports of the target application, the accessibility to which is impacted </td>
+        <td> comma separated ports(S) can be provided. If not provided, it will induce network chaos for all ports</td>
```

Comment:
Can we make this change in all the places?
```suggestion
        <td> comma separated port(s) can be provided. If not provided, it will induce network chaos for all ports</td>
```